lib/drivelist.js

```diff
@@ -47,6 +48,11 @@ const scripts = require('./scripts.json');
  */
 exports.list = (callback) => {
   const operatingSystem = os.platform();
+
+  if (operatingSystem == 'win32') {
```

Comment:
Probably want to use strict equality `===` here to please the eslint overlord ;)